src/serializer/serializer.js

```diff
@@ -1748,6 +1833,6 @@ export class Serializer {
     let serialized = this.serialize(filename, code, sourceMaps);
     invariant(!this.logger.hasErrors());
     if (this.options.logStatistics) this.statistics.log();
+    serialized.generated.statistics = this.statistics;
```

Comment:
Updated, thanks for the hints!  I added this line so I could test against the output of `statistics`, any ideas about the right way to get access to this from `test-runner`?